io.cpp

```diff
@@ -37,12 +37,54 @@ void setupRadio()
     }
     Serial.print(F("FREQ: ")); Serial.println(config.rf95_freq);
     rf95.setTxPower(config.tx_power, false);
-    rf95.setCADTimeout(2000);
+    //rf95.setCADTimeout(2000);
```

Comment:
I think we need the CADTimeout in here. If it is not working for you, I think we need to carefully inspect the RadioHead versions. I might have had to use a version of RadioHead cloned from the repository rather than downloaded from the site. I'm not quite sure

Reply:
I get the following error when I include it:
```
io.cpp:40: error: 'class RH_RF95' has no member named 'setCADTimeout'
```